corehq/apps/app_manager/xform.py

```diff
@@ -972,24 +972,19 @@ def get_questions(self, langs, include_triggers=False,
         """
         from corehq.apps.app_manager.util import first_elem
 
-        def _add_choices_for_select_questions(question):
-            if cnode.items is not None:
-                options = []
-                for item in cnode.items:
-                    translation = self._get_label_text(item, langs)
-                    try:
-                        value = item.findtext('{f}value').strip()
-                    except AttributeError:
-                        raise XFormException(_("<item> ({}) has no <value>").format(translation))
-                    option = {
-                        'label': translation,
-                        'value': value
-                    }
-                    if include_translations:
-                        option['translations'] = self._get_label_translations(item, langs)
-                    options.append(option)
-                question['options'] = options
-            return question
+        def _get_select_question_option(item):
```

Comment:
This change might be clearer with https://github.com/dimagi/commcare-hq/pull/22664/commits/db894bf0594a7a872683b8d57b09f7efcffbb469?w=1